provider/src/main/scala/com/actian/spark_vector/provider/RequestHandler.scala

```diff
@@ -66,27 +65,9 @@ class RequestHandler(sqlContext: SQLContext, val auth: ProviderAuth) extends Log
     var jobPartAccum = Future { () }
```

Comment:
nit: I know it's not your code, but if `Future()` works here instead of `Future { () }` please replace it :v: